apps/api/src/services/competition-manager.service.ts

```diff
@@ -188,6 +178,15 @@ export class CompetitionManager {
       );
     }
 
+    // Always create trading constraints (with defaults if not provided)
+    await this.tradingConstraintsService.createConstraints({
+      competitionId: id,
+      ...tradingConstraints,
```

Comment:
this doesnt create with the defaults, though, right? it uses the param passed by the method, which would come from the admin API. 

i.e., either in this method or the [admin method](https://github.com/recallnet/js-recall/blob/9722600dcd55a6391f8a428a5e11e141f084d3bd/apps/api/src/controllers/admin.controller.ts#L598), we should check if `tradingConstraints` are provided. and if not, we would get defaults from the config and then use those as the values.

Reply:
also, i think we should include the `tradingConstraints` in the response for `startCompetition` and `createCompetition`. that'll help ensure it's enforced.